tests/unit/test_pipeline_graph.py

```diff
@@ -172,48 +174,91 @@ def test_update_slot_filepaths(default_config: Config, test_dir: str) -> None:
         assert edge_attrs["filepaths"] == expected_filepaths[(source, sink)]
 
 
-def test_get_input_slots(default_config: Config, test_dir: str) -> None:
+def test_get_io_slots(default_config: Config, test_dir: str) -> None:
```

Comment:
`get_input_slots` became `get_io_slots` b/c we now need output slot data just like input slot data.